kibana-reports/server/routes/lib/createReport.ts

```diff
@@ -74,6 +81,7 @@ export const createReport = async (
       );
     } else {
       // report source can only be one of [saved search, visualization, dashboard]
+      // Check if security is enabled. TODO: is there a better way to check?
```

Comment:
Whats the current check for if security is enabled? the `if` block with the `request.headers.cookie` object?

Reply:
From my test, if security is not enabled, `request.header.cookie` will be `undefined`